homeassistant/components/zha/core/gateway.py

```diff
@@ -672,7 +672,7 @@ async def async_create_zigpy_group(
     async def async_remove_zigpy_group(self, group_id: int) -> None:
         """Remove a Zigbee group from Zigpy."""
         if not (group := self.groups.get(group_id)):
-            _LOGGER.debug("Group: %s:0x%04x could not be found", group.name, group_id)
+            _LOGGER.debug("Group: 0x%04x could not be found", group_id)
```

Comment:
Unless I am mistaken, according to the line above group is always `None` in this case, so you can't possibly extract the name out of it!